src/PrestaShopBundle/Form/Admin/Sell/Customer/CustomerType.php

```diff
@@ -358,6 +383,20 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 ])
             ;
         }
+
+        // We add a listener that will make password field not required, if we want to create a guest
+        $builder->addEventListener(FormEvents::PRE_SUBMIT, function (FormEvent $event) {
```

Comment:
If you are motivated, you can separate this to another class, here's an example:
https://github.com/PrestaShop/PrestaShop/pull/31568/files

I'm sure @zuk3975 would appreciate it :D

Reply:
Maybe next time, I will do something useful instead 🥲